presto-proxy/pom.xml

```diff
@@ -170,6 +170,12 @@
         <dependency>
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-main</artifactId>
+            <exclusions>
```

Comment:
Added exclusion because H3 classes were causing duplicate classes
```
[WARNING] Found duplicate (but equal) resources in [com.sun.istack:istack-commons-runtime:3.0.7, com.uber:h3:4.0.0]:
[WARNING]   .gitkeep
[WARNING] Found duplicate classes/resources in test classpath.
```
Also presto-main is only used in <scope> test